tests/markdown-data/emphasis.html

```diff
@@ -12,5 +12,6 @@
 <p><em>emphasized text</em><strong>strong text</strong></p>
 <p><strong>strong text</strong><em>emphasized text</em></p>
 <p><em>emphasized text</em><strong>strong text</strong></p>
+<p><strong>strong text A</strong><strong>strong text B</strong></p>
```

Comment:
Yeah I see now, thank you. Will think a bit about it. Not breaking other tests does not mean it does not break anything else, will check that out in more detail later :)